distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -288,6 +290,22 @@ meta value = if is_atom value then Atom.Value value else
                 if is_error value then Error.Value value.catch else
                     Primitive.Value value
 
+## Given a type object, method name and parameter name, return the associated Attribute if it exists.
+
+   Attributes:
+   - target: The value or type to get the attribute from.
+   - method_name: The name of the method to get the attribute for.
+   - parameter_name: The name of the parameter to get the attribute for.
+Meta.get_annotation : Any -> Text -> Text -> Any | Nothing
+Meta.get_annotation target method_name parameter_name =
+    mock_table = if (target.to_text.contains "org.enso.table.data.table.Table@").not then Nothing else case method_name of
+        "at" -> case parameter_name of
+            "selector" -> target.columns.map .name
```

Comment:
The specification requests:
```
@field Single_Choice display=Display.Always values=(self.columns.map .name)
Table.at : (Text | Integer) -> Column
Table.at self field=(self.columns.at 0 . name) = ...
```
e.g. there should be `Single_Choice` and `Display` somewhere. @jdunkerley can you put them at (some) appropriate place in this PR? Then we can start returning them instead of plan `Vector Text`.

Reply:
Have added the types to `Standard.Base.Metadata`. Haven't tested the code but think should work!

Reply:
> Metadata.enso[21:37-21:40]: The name `code` could not be found.

Corrected in [d315858](https://github.com/enso-org/enso/pull/3986/commits/d315858bcfcb371afb09b411f10380dec64da673)